incubator/crate/values.yaml

```diff
@@ -0,0 +1,53 @@
+# The Chart for a crateDB deployment with working defaults consists of:
+#   1) A Service (exposing the ports that crate uses)
+#   2) A StatefulSet (initialising the cluster)
+#   3) Resources definition
+app: crate
+
+image:
+  name: crate
+  tag: 4.0.3
+  pullPolicy: IfNotPresent
+
+# 1) The Service connecting 3 ports.
+service:
+  name: crate
+  type: ClusterIP
+
+  adminName: crate-web
+  adminPort: 4200
+  clusterName: cluster
+  clusterPort: 4300
+  # https://crate.io/docs/crate/reference/en/latest/interfaces/postgres.html
+  postgresName: postgres
+  postgresPort: 5432
```

Comment:
Agreed, done. Only reason I can think of for configuring `transport` differently would be because for some reason the UI/psql port is really wanted to be 4300... but that's mad, as above let's keep it simpler and more prescriptive for now at least.